### ISSUE TITLE
Web interface unavailable

### ISSUE THREAD

**Issue by bjarneo** [meilisearch/MeiliSearch#479]
Tested using the docker image. This is the result:
```
http -h http://127.0.0.1:7700 
HTTP/1.1 404 Not Found
access-control-allow-origin: *
content-encoding: gzip
date: Tue, 25 Feb 2020 11:11:48 GMT
transfer-encoding: chunked
```

However, I am able to use the REST API though.

**Comment by tpayet** [comment]
Hello :)
I suppose your are using the latest tag with docker, which is the latest stable tag.

The web interface is available since the v0.8.5-rc.2, which you can use with docker : `docker run --rm -p 7700:7700 getmeili/meilisearch:v0.8.5-rc.2`. Please let me know if this works for you.

We are working on releasing a stable v0.8.5 asap

**Comment by bjarneo** [comment]
O, right. I'll try the latest version. Thanks for the response :)
